src/docs/background/16-how-saving-payment-card-works-in-ftw/index.md

```diff
@@ -0,0 +1,135 @@
+
+## Saving card details
+
+There are 2 ways to save payment card details:
+
+- saving cards details without making an initial payment
+- saving cards details after a payment
+
+The former of those can be done in payment methods page (profile menu ->
+account settings -> payment methods), but you can also save payment card
+details on the checkout page when making a one-time payment.
+
+> Note: currently, it is only possible to save one card, which becomes
+> the default payment method. I.e. if there is already one payment card
+> saved, your only option is to replace the card with a new one.
+
+### Saving cards without making an initial payment
+
+If the user has no existing default payment card, `PaymentMethodsPage`
+component will show just a form to save payment card details:
+
+![Payment methods page](payment-methods-page.png 'Payment methods page without default payment method.')
+
+If there is a default payment methods saved, user will see a dropdown
+instead, where the current default method is selected. There are two
+options the user can do: replace the current payment card (it's an
+option inside the dropdown) or delete the current payment card.
+
+![Payment methods page saved card](payment-methods-page-saved-card.png 'Payment methods page with default payment method.')
+
+Under the hood, saving a new payment method needs to complete following
+steps:
+
+- Create Stripe
+  [Setup Intent](https://stripe.com/docs/payments/cards/saving-cards#create-setup-intent)
+  through Flex API to obtain the client secret.
+- Call `stripe.handleCardSetup` with the client secret
+  - `stripe.handleCardSetup` will handle user actions like 3D Secure
+    authentication
+- Save payment method:
+  - Create a new Stripe Customer if needed and attach the new payment
+    method to `stripeCustomer` entity.
+  - There's no replace or update option for `defaultPaymentMethod`
+    entity. To replace a payment card, one needs to remove previous card
+    first and then add a new payment method.
+
+### Saving cards after a payment
+
+If the user doesn't have a default payment card saved, or if she is
+making a one-time payment with a different payment card, there's an
+option (checkbox) to save the new payment card as the default payment
+method.
+
+![One-time payment and saving card details for future use](one-time-payment.png 'One-time payment and saving card details for future use')
+
+If the default payment method exists, user needs to select one-time
+payment first and then she is able to replace the default card.
+
+![Pay with one-time payment card](pay-with-new-one-time-card.png 'Pay with one-time payment card')
+
+**Under the hood, there are few changes made to the Payment Intents
+checkout flow:**
+
+- We need to know if the user has already saved a default payment
+  method:
+  - Fetch `currentUser` entity with
+    `stripeCustomer.defaultPaymentMethod` relationship
+- Show correct form fields:
```

Comment:
Even though it's kind of mentioned in the screenshots it might be more clear to explain all the possible options for UI: when there is no default card, show just form and checkbox, if there is default card show the SavedCardComponent and if user wants to use another card, show the form and option to replace card.